insights/parsers/kpatch_patches.py

```diff
@@ -0,0 +1,40 @@
+"""
+KpatchPatches - report locally stored kpatch patches
+====================================================
+
+This parser creates a list of the module names of locally
+stored kpatch modules. If no modules are installed, the
+list will be empty.
+
+"""
+
+from .. import parser, Parser
+from insights.specs import Specs
+import re
+
+
+@parser(Specs.kpatch_patch_files)
+class KpatchPatches(Parser):
```

Comment:
This parser is for a command, better change to `CommandParser` to get the best usage.
See https://github.com/RedHatInsights/insights-core/blob/34505e28187c71c364a7c463d3f236caba192c7d/insights/core/__init__.py#L511-L523

Reply:
Changed in latest patch